easypost/webhook.py

```diff
@@ -17,3 +27,30 @@ def update(self, **params) -> "Webhook":
         response, api_key = requestor.request(method=RequestMethod.PATCH, url=url, params=params)
         self.refresh_from(values=response, api_key=api_key)
         return self
+
+    @staticmethod
+    def validate_webhook(webhook_body: Dict[str, Any], headers: Dict[str, Any], webhook_secret: str) -> bool:
```

Comment:
you can't validate over a dict, you can only validate over a series of bytes, because JSON doesn't have a canonical serialization.

Reply:
Ah yes, had this in the function when really this should have happened in the fixture and passed into the function. Corrected.